ci/scripts/cpp_build.sh

```diff
@@ -129,6 +129,10 @@ time cmake --build . --target install
 
 popd
 
+if [ -x "$(command -v ldconfig)" ]; then
+  ldconfig
```

Comment:
Note that this won't work if non-root.

Reply:
In docker we act as root, at least for now.